modules/tensor_mechanics/contrib/neml2.mk

```diff
@@ -0,0 +1,101 @@
+# There are 2 influential conf vars.
+# They are controlled by the MOOSE configure script.
+#
+# ENABLE_LIBTORCH: True if we should compile with libTorch AND an installation
+#                  of libTorch is available. False otherwise.
+# LIBTORCH_DIR:    The path to the libTorch installation. NEML2 will use this
+#                  libTorch as the backend.
+#
+# There are 2 influential env vars.
+#
+# ENABLE_NEML2:  If true, BlackBear will attempt to compile with NEML2 at
```

Comment:
look for references to blackbear . I get a lot of hits on ctrl F

Reply:
I think I got them all now